Lib/test/test_capi/test_opt.py

```diff
@@ -1564,6 +1564,23 @@ def f(n):
         # But all of the appends we care about are still there:
         self.assertEqual(uops.count("_CALL_LIST_APPEND"), len("ABCDEFG"))
 
+def test_compare_pop_two_load_const_inline_borrow(self):
```

Comment:
Forgot to indent this line. :)

Reply:
Ah, I was matching the test above...however, that also has the wrong indentation. Will fix both!

Reply:
Ah, thanks! I think that was the result of a bad merge.